packages/contracts/script/utils/exporter.js

```diff
@@ -3,19 +3,31 @@ require('dotenv').config();
 const path = require('path');
 const fs = require('fs');
 const { createClient } = require('@supabase/supabase-js');
+const { load } = require('js-toml');
+
+// Loads the environment-specified TOML configuration file.
+async function loadConfiguration() {
+  const env = process.env.ENVIRONMENT;
+  const configFile = path.resolve(__dirname, `../../resource/${env}.toml`);
+  console.log(`Loading configuration from: '${configFile}'`);
+  const toml = fs.readFileSync(configFile, 'utf8');
+  return load(toml);
+}
+
+async function exportAddress(config) {
```

Comment:
looks good!  is config here typed meaning if the toml changes we can a compilation error?  if not, do we want to have a typed version for ourselves to quickly spot issues where the toml file and code are out of sync?

Reply:
No `config` is not typed. I thought it should be and looked it up. Seeing as the code is JS, not type possible.